Web/View/ParagraphQuotes/Show.hs

```diff
@@ -20,13 +21,17 @@ instance View ShowView where
                             , breadcrumbText "Show ParagraphQuote"
                             ]
 
-renderParagraph :: Text -> Text -> Text -> Html
-renderParagraph body subtitle imageUrl =
+-- @todo: Find the right type signature.
+renderParagraph :: Text -> Text -> Text -> IO Html
```

Comment:
This is causing a compile error

![image](https://github.com/amitaibu/ihp-landing-page/assets/125707/23ab8d24-e055-4b2d-a1a5-89a3b00a0bde)